### ISSUE TITLE
Feature Request: Load Quiz Questions from a File

### ISSUE THREAD

**Issue by Magnoir** [KryMucus/-Quiz_Game---Jules_Deleuse---CDOF3-#6]
## Description

Currently, the quiz questions are hardcoded into the Python script. This limits the flexibility of the quiz and makes it harder to add, remove, or change questions. 

## Suggested Feature

I propose we add a feature to load the quiz questions from a text file. Each question could be a line in the file, with the question, answers, and correct answer separated by some delimiter. Here's an example of how we could structure the file:

**Comment by KryMucus** [comment]
Hello @Magnoir, this is a good idea ! It could be interesting to extand it to  .csv files too ! Do you want to get started on that? I can Assign it to you right now, if so !

**Comment by analytical-aakriti** [comment]
Storing it in CSV is a great idea. I can take it up if you want

**Comment by KryMucus** [comment]
Yes please, can you tell us if you want to do both the .csv, and .txt, loading, or just the .csv ? @analytical-aakriti 

Are you still interested @Magnoir ?

**Comment by analytical-aakriti** [comment]
I have done it from .csv here https://github.com/KryMucus/Quiz_Game---Jules_Deleuse---CDOF3-/pull/11

**Comment by Magnoir** [comment]
I can do it in .txt

**Comment by Magnoir** [comment]
I added it, here is the proof 
![image](https://github.com/KryMucus/Quiz_Game---Jules_Deleuse---CDOF3-/assets/44030687/e53e532a-6901-4b04-b5e2-68eea75707ef)

**Comment by KryMucus** [comment]
> I added it, here is the proof 
> 
> ![image](https://github.com/KryMucus/Quiz_Game---Jules_Deleuse---CDOF3-/assets/44030687/e53e532a-6901-4b04-b5e2-68eea75707ef)
> 
> 

Have you added the file to your commit ? I don't see it in your pull request.

**Comment by Magnoir** [comment]
I will look for it
